libmscore/slur.cpp

```diff
@@ -42,20 +46,22 @@ void SlurSegment::draw(QPainter* painter) const
                   break;
             case 1:
                   painter->setBrush(Qt::NoBrush);
+                  pen.setCapStyle(Qt::RoundCap); // round dots
                   pen.setWidthF(score()->styleP(Sid::SlurDottedWidth) * mag);
-                  pen.setStyle(Qt::DotLine);
-                  break;
+                  pen.setStyle(Qt::CustomDashLine);
+                  pen.setDashPattern(dotted);
+                  break;     
             case 2:
                   painter->setBrush(Qt::NoBrush);
                   pen.setWidthF(score()->styleP(Sid::SlurDottedWidth) * mag);
-                  pen.setStyle(Qt::DashLine);
+                  pen.setStyle(Qt::CustomDashLine);
```

Comment:
same

Reply:
Removed those lines. Also reordered for clarity on all instances (styles first, width on the end like default slurs case 0)